### ISSUE TITLE
Let athletes manage their bookings using their email or phone number

### ISSUE THREAD

**Issue by silviot** [eisbuk/EisBuk#383]
Depends on #382 and #381

The non-authorized view is currently displayed to non-admin logged-in users.
We want instead to look up if there's an athlete with that phone number or email, and if there is, redirect users to the secret-containing-url to manage their bookings. 

If no user is found the unauthorized page should be shown, but should be changed to read something like this:
```
The email $EMAIL/phone numeber $PHONE were not found in our registry.
Please try a different one or get in touch at $EMAIL_FROM
```

**Comment by ikusteu** [comment]
Closing this as it's mainly done and the bug fixes are included as a separate issue #419
